.github/workflows/integration.yml

```diff
@@ -26,14 +27,39 @@ jobs:
         sudo apt-get update
         sudo apt-get install g++-9 libsodium-dev libsqlite3-dev
 
+    - name: Checkout faucet
+      uses: actions/checkout@v2
+      with:
+        repository: bosagora/faucet
```

Comment:
So it turns out that actually checking out faucet instead of agora works better, who would've guessed.

Reply:
Hmm, so it was not checking out without the `repository`?

Reply:
It is checking out Agora (again)